src/domain/notifications/notifications.repository.v2.ts

```diff
@@ -63,7 +66,13 @@ export class NotificationsRepositoryV2 implements INotificationsRepositoryV2 {
       );
     } catch (e) {
       if (this.isTokenUnregistered(e)) {
-        await this.deleteDevice(args.deviceUuid).catch(() => null);
+        this.loggingService.info(
+          `Deleting unregistered token for device ${args.deviceUuid}: ${e}`,
+        );
```

Comment:
Maybe I'm not getting it right, but this message is stating we are deleting an unregistered token, and actually the device is being deleted, isn't it? (according to the subsequent datasource function call)

Reply:
I adjusted the message in 2a1216101fd50bafd404c24e2a6fddae907dcbf4 to explain that we're deleting a device _because of_ the token being stale.